### PR TITLE
Updated C# Dev Kit FAQ

### PR DESCRIPTION
**Pull request by mikadumont**

### CODE REVIEW

File: docs/csharp/cs-dev-kit-faq.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Use this FAQ (Frequently Asked Questions) topic to learn more about the C# Dev K
...
+C# Dev Kit supports building web apps, console apps, class library projects, and test projects for .NET Core, often also called .NET. The [.NET MAUI extension](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.dotnet-maui) and the [Unity extension](https://marketplace.visualstudio.com/items?itemName=visualstudiotoolsforunity.vstuc) are built on top of C# Dev Kit and provide additional support for building [.NET Multi-platform App UI (MAUI)](https://dotnet.microsoft.com/apps/maui) apps and Unity apps. These extensions support the modern .NET project format, also known as "sdk-style" projects. If you are building non-SDK-format projects such as .NET Framework apps and Xamarin apps, see the Project System section.
```
Comment (webreidi) [inline docs/csharp/cs-dev-kit-faq.md:23]: Add link to https://learn.microsoft.com/en-us/dotnet/standard/choosing-core-framework-server on .NET Framework.

File: docs/csharp/cs-dev-kit-faq.md [hunk 2]
```diff
@@ -103,7 +123,7 @@ This is usually because the project targets .NET Framework rather than .NET Core
 
 ![Project Not Supported in Solution Explorer](images/faq/solution-explorer-not-supported-framework.png)
 
-You will need to [update your project](https://learn.microsoft.com/dotnet/core/porting/) to an SDK-style project to use all available C# Dev Kit features.
+You can either [update your project](https://learn.microsoft.com/en-us/dotnet/core/porting/) to an SDK-style project to use all available C# Dev Kit features. Or you can use the new Prefer CSharp Extension workspace setting located in the Settings UI. This setting delegates solution and project load to the C# extension but C# Dev Kit features will be unavailable.
```
Comment (webreidi) [inline docs/csharp/cs-dev-kit-faq.md:126]: This needs more details. Tell them how to set the setting (both options, with the toast or manually).

**Review by webreidi — CHANGES_REQUESTED** [review]
Looks good. Just a couple of changes.

File: docs/csharp/cs-dev-kit-faq.md [hunk 3]
```diff
@@ -103,7 +123,9 @@ This is usually because the project targets .NET Framework rather than .NET Core
 
 ![Project Not Supported in Solution Explorer](images/faq/solution-explorer-not-supported-framework.png)
 
-You will need to [update your project](https://learn.microsoft.com/dotnet/core/porting/) to an SDK-style project to use all available C# Dev Kit features.
+To resolve this issue, you have two options. You can [update your project](https://learn.microsoft.com/en-us/dotnet/core/porting/) to an SDK-style project to access all available C# Dev Kit features.
+
+Alternatively, you can use the new "Prefer CSharp Extension" workspace setting located in the Settings UI. This setting delegates solution and project load to the C# extension, but some C# Dev Kit features will be unavailable. If you have a .NET Framework project C# Dev Kit will show a toast asking you to either update your project to an SDK-style project or have the C# extension load your project or solution.
```
Comment (webreidi) [inline docs/csharp/cs-dev-kit-faq.md:128]: I think you should also mention about needing to reload once the setting has been applied for it to take effect.

**Review by webreidi — APPROVED** [review]

File: docs/csharp/cs-dev-kit-faq.md [hunk 4]
```diff
@@ -71,6 +71,16 @@ Note that, depending on your network speed, installing the .NET Core runtime mig
 }
 ```
 
+### Error Acquiring .NET 7
+
+![Error Acquiring .NET 7](images/faq/error-acquiring-dotnet-7.png)
+
+Note: If you are located in China the Great Firewall of China may be blocking your .NET download which can cause a timeout.
```
Comment (gregvanl) [inline docs/csharp/cs-dev-kit-faq.md:78]: Are we allowed to say "Great Firewall of China"?
Might be better to say "If you are located in China, your .NET download may be blocked and cause a timeout."

File: docs/csharp/cs-dev-kit-faq.md [hunk 5]
```diff
@@ -127,7 +151,9 @@ The most likely cause of this error is a `global.json` file that specifies a dif
 
 ![A compatible .NET SDK was not found](images/faq/compatible-dotnet-sdk-not-found.png)
 
-Open the Output window (`kb(workbench.action.output.toggleOutput)`) and switch to the **Projects** pane to look for more information.
+Open the Output window and switch to the “Projects” pane to look for more information. You should see something like this:
```
Comment (gregvanl) [inline docs/csharp/cs-dev-kit-faq.md:154]: Use bold for UI elements so keep `**Projects**`

File: docs/csharp/cs-dev-kit-faq.md [hunk 6]
```diff
@@ -71,6 +71,16 @@ Note that, depending on your network speed, installing the .NET Core runtime mig
...
 
+### Error Acquiring .NET 7
+
+![Error Acquiring .NET 7](images/faq/error-acquiring-dotnet-7.png)
+
+Note: If you are located in China the Great Firewall of China may be blocking your .NET download which can cause a timeout.
+
+You will need to make sure you have .NET 7 installed. As a workaround, you can point the .NET runtime acquisition extension to an existing .NET 7 install:
+
+![Point the .NET runtime acquisition extension to an existing .NET 7 install](images/faq/point-dotnet-runtime-to-existing-dotnet-install.png)
```
Comment (gregvanl) [inline docs/csharp/cs-dev-kit-faq.md:82]: Better to use a code block for the settings JSON example than a screenshot.
That way users can easily copy and paste to get started

**Review by gregvanl — APPROVED** [review]

**Pull request merged**